gdal/alg/gdalwarper.h

```diff
@@ -61,7 +61,8 @@ typedef enum {
   /*! Med (selects median of all non-NODATA contributing pixels) */ GRA_Med=10,
   /*! Q1 (selects first quartile of all non-NODATA contributing pixels) */ GRA_Q1=11,
   /*! Q3 (selects third quartile of all non-NODATA contributing pixels) */ GRA_Q3=12,
-  /*! Sum (computes the weighed sum of all non-NODATA contributing pixels). Added in GDAL 3.1 */ GRA_Sum=13
+  /*! Sum (computes the weighed sum of all non-NODATA contributing pixels). Added in GDAL 3.1 */ GRA_Sum=13,
+  /*! Average (computes the weighted quadratic average of all non-NODATA contributing pixels) */ GRA_AverageQuadratic=14
```

Comment:
maybe add the word RMS in the comment since it is prefered by other people ?

Reply:
May be it should me more clear if i rename all AVERAGE_QUADRATIC with RMS?

Reply:
> May be it should me more clear if i rename all AVERAGE_QUADRATIC with RMS?

I'm fine with that. I see in https://en.wikipedia.org/wiki/Root_mean_square that "quadratic mean" seems to be an accepted term, so maybe in the doc of the enum mentions it too

Reply:
Done